cmd/setup-etcd-environment/run.go

```diff
@@ -30,13 +30,16 @@ var (
 		discoverySRV string
 		ifName       string
 		outputFile   string
+		bootstrapSRV bool
```

Comment:
Where does the new flag --bootstrapSRV get used?

Reply:
it doesn't by default

Reply:
But during the recovery this should be set to false, right?

Reply:
I think it should be a general option to disable but the correct way to change the spec would be via MachineConfig. The downside of that update would be a rolling restart of nodes, not great during discovery. For DR we will handle this with sed, after the cluster is restored and stable the ENV could be permanently disabled with MachineConfig update.

Reply:
Yeah. On reboot the environment gets reset, so we should be fine.